services/github/index.ts

```diff
@@ -0,0 +1,46 @@
+import simpleGit, { SimpleGit } from "simple-git";
+import fs from "fs/promises";
+import path from "path";
+import config from "./config.json";
+
+const { owner, repo, ssh } = config;
+
+const { GH_ACCESS_TOKEN } = process.env;
+
+interface WikiParams {
+  wikiPage: string;
+  wikiContent: string;
+}
+
+export default async function updateWiki(params: WikiParams): Promise<void> {
+  try {
+    const git: SimpleGit = simpleGit();
+
+    // Clone the wiki
+    const wikiUrl: string = ssh
+      ? `git@github.com:${owner}/${repo}.wiki.git`
+      : `https://${GH_ACCESS_TOKEN}@github.com/${owner}/${repo}.wiki.git`;
+
+    await git.clone(wikiUrl, `./wikis/${repo}`);
+
+    // Create or edit the wiki page
+    await fs.writeFile(
+      path.join(`./wikis/${repo}`, `${params.wikiPage}.md`),
+      params.wikiContent
+    );
+
+    // Commit and push the changes
+    await git.cwd(`./wikis/${repo}`);
+    await git.add("./*");
+    await git.commit("Updated the wiki");
+    await git.push("origin", "master");
```

Comment:
I assume that `"main"` should be here. Does it make a difference thought?

Reply:
I dont think it does the same, I did not even recognize it, I will try later to change it.